### PR TITLE
Upgrade deps

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

- [X] upgrade dependencies to fix CVE
- [X] address linting

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: connector.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func (c *connector) Deploy(ctx context.Context, image string) (deployer.Plugin,
 
 func (c *connector) startContainer(ctx context.Context, cnt *connectorContainer) error {
 	c.logger.Debugf("Starting container %s...", cnt.id)
-	if err := c.cli.ContainerStart(ctx, cnt.id, types.ContainerStartOptions{}); err != nil {
+	if err := c.cli.ContainerStart(ctx, cnt.id, container.StartOptions{}); err != nil {
```
Comment (dbutenhof) [inline connector.go:58]: So docker v25 has incompatible changes from v24, but the changes are restricted to the names of these three referenced types? Interesting ... I hope the CI tests this well!

**Review by dbutenhof — APPROVED** [review]

File: container.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package docker
 import (
 	"context"
 	"fmt"
+	"github.com/docker/docker/api/types/container"
 	"time"
 
 	"github.com/docker/docker/api/types"
```
Comment (webbnh) [inline container.go]: Are these supposed to be ordered?  (What is the blank line for, and why do we have `github.com` references on both sides of it?)

**Review by webbnh — APPROVED** [review]
Just one pointed question.

File: container.go (hunk 2 above)
Comment (mfleader) [reply container.go]: That's odd. I thought the formatter would order them

Comment (mfleader) [reply container.go]: They're formatted now.

File: connector.go (hunk 1 above)
Comment (mfleader) [reply connector.go:58]: me too

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**